net/ngtcp2/Portfile

```diff
@@ -34,6 +30,12 @@ configure.args-append   --disable-silent-rules \
                         --enable-werror=no \
                         --with-gnutls=yes
 
+variant openssl description "Enable OpenSSL" {
+    depends_lib-append  port:openssl3
```

Comment:
This should not be here, PortGroup already adds the dependency.

Reply:
Why though? If you want to use `openssl3`, just keep the PG. If configure does not find it, then check the portgroup file for related variables.